packages/server/src/core/TeamSpeak/index.ts

```diff
@@ -100,7 +102,7 @@ function listenEvents() {
 /**
  * Before exit
  */
-function beforeExit() {
+function beforeExit(): void {
```

Comment:
This may cause a bug where if you CTRL-C the running bot it would stop but for some reason continuing the execution. Can't replicate it...

Reply:
Cant replicate it on local. For now this issue is resolved.